vagrant/backend-v1/app/install.sh

```diff
@@ -3,18 +3,18 @@
 set -e
 
 cat << "EOF"
-oooooooooo.        .o.         .oooooo.   oooo    oooo oooooooooooo ooooo      ooo oooooooooo.   
-`888'   `Y8b      .888.       d8P'  `Y8b  `888   .8P'  `888'     `8 `888b.     `8' `888'   `Y8b  
- 888     888     .8"888.     888           888  d8'     888          8 `88b.    8   888      888 
- 888oooo888'    .8' `888.    888           88888[       888oooo8     8   `88b.  8   888      888 
- 888    `88b   .88ooo8888.   888           888`88b.     888    "     8     `88b.8   888      888 
- 888    .88P  .8'     `888.  `88b    ooo   888  `88b.   888       o  8       `888   888     d88' 
-o888bood8P'  o88o     o8888o  `Y8bood8P'  o888o  o888o o888ooooood8 o8o        `8  o888bood8P'   
+
+ ____    _    ____ _  _______ _   _ ____   __     __  _ 
+| __ )  / \  / ___| |/ / ____| \ | |  _ \  \ \   / / / |
+|  _ \ / _ \| |   | ' /|  _| |  \| | | | |  \ \ / /  | |
+| |_) / ___ \ |___| . \| |___| |\  | |_| |   \ V /   | |
+|____/_/   \_\____|_|\_\_____|_| \_|____/     \_/    |_|
+                                                        
```

Comment:
can you drop a comment somewhere how did you generate this? When I was adding prometheus with grafana I couldn't find it.

Reply:
added note on all ascii art